test/viewsapi/FormFunctionSpec.scala

```diff
@@ -60,21 +60,37 @@ class FormFunctionSpec extends FrontEndTestHelper {
       result shouldBe ""
     }
 
-    "not render an asterisk when no '_requiredOption" in {
+    "not render an asterisk when no _requiredOption" in {
       val args: Map[Symbol, Any] = Map(Symbol("_madeUpOption") -> true)
 
       val result = FormFunctions.inputRenderOptions(args).requiredLabelSuffix()
       result shouldBe ""
     }
   }
 
+  "disabledInput function" should {
+    "render disabled when _disabledOption set to a particular input option" in {
+      val args: Map[Symbol, Any] = Map(Symbol("_disabledOption") -> "false")
+
+      val result = FormFunctions.inputRenderOptions(args).disabledInput("false")
+      result shouldBe Some("disabled")
+    }
+
+    "not render disabled when no _disabledOption" in {
+      val args: Map[Symbol, Any] = Map(Symbol("_madeUpOption") -> "true")
+
+      val result = FormFunctions.inputRenderOptions(args).disabledInput("true")
+      result shouldBe ()
+    }
+
   "setErrorClass function" should {
```

Comment:
There's a new `selectedInput` method in the form functions but there's no test for these in here.

Reply:
fixed